game/engine/src/main/kotlin/de/gleex/pltcmd/game/engine/systems/facets/Detects.kt

```diff
@@ -64,16 +66,29 @@ object Detects : BaseFacet<GameContext, DetectEntities>(
     private fun logSeen(
         seeing: SeeingEntity,
         seen: PositionableEntity,
-        visibility: SignalStrength,
-        previousVisibility: Visibility
+        visibility: SignalStrength
     ) {
         if (log.isDebugEnabled()) {
             val who = seeing.logIdentifier.padEnd(12)
             val what = seen.logIdentifier.padEnd(12)
             val where = seen.currentPosition.toString().padEnd(12)
             val how = visibility.asRatio()
-            log.debug("$who sees $what at $where with signal strength $how (last visibility was $previousVisibility)")
+            log.debug("$who sees $what at $where with signal strength $how")
```

Comment:
`visiblility` is actually a `Visibility`. It is semantically more accurate and would also more oviously lead to a more descriptive log message:
```suggestion
            log.debug("$who sees $what at $where with visibility $how")
```

Reply:
> `visiblility` is actually a `Visibility`. 

Nope, it is a `SignalStrength`. For debugging the actual value rather then the category (GOOD/POOR/NONE) is more interesting.